### PR TITLE
Update automl zouwu doc

### PR DESCRIPTION
**Pull request by yushan111**
main changes:
1. change ray version to 0.7.2 in [zouwu tutorial].(https://analytics-zoo.github.io/master/#Zouwu/tutorial/)
2. fix format in [zouwu overview](https://analytics-zoo.github.io/master/#Zouwu/overview/)
3. add reference for network traffic ReadMe and remove requirements.
4. remove main docs in automl Readme.

### CODE REVIEW

**Comment by yushan111** [comment]
PR passed in http://10.239.47.210:18888/job/ZOO-PR-Doc-Generation-Test/358/

File: pyzoo/zoo/zouwu/use-case/network_traffic/README.md [hunk 1]
```diff
@@ -19,29 +19,11 @@ ie. LSTM and MTNet, to do time series forecasting. Both univariate and multivari
...
 
-We recommend you to use [Anaconda](https://www.anaconda.com/distribution/#linux) to prepare the enviroments.
-```
-conda create -n zoo python=3.6 #zoo is conda enviroment name, you can set another name you like.
-source activate zoo
-pip install analytics-zoo[automl]
-```
-Note that the extra dependencies (including `ray`, `psutil`, `aiohttp`, `setproctitle`, `scikit-learn`,`featuretools`, `tensorflow`, `pandas`, `requests`, `bayesian-optimization`) will be installed by specifying `[automl]`.  
-You can also find the details of zoo installation [here](https://analytics-zoo.github.io/master/#PythonUserGuide/install/) 
+You can refer to zouwu installation document [here](https://analytics-zoo.github.io/master/#Zouwu/tutorial/#install).
```
Comment (jason-dai) [inline pyzoo/zoo/zouwu/use-case/network_traffic/README.md:26]: Zouwu

File: pyzoo/zoo/zouwu/use-case/network_traffic/README.md [hunk 2]
```diff
@@ -19,29 +19,11 @@ ie. LSTM and MTNet, to do time series forecasting. Both univariate and multivari
 demonstrated in the example.
```
Comment (shane-huang) [inline pyzoo/zoo/zouwu/use-case/network_traffic/README.md:20]: Remove #Introduction section title in README

File: pyzoo/zoo/zouwu/use-case/network_traffic/README.md [hunk 3]
```diff
@@ -19,29 +19,11 @@ ie. LSTM and MTNet, to do time series forecasting. Both univariate and multivari
...
-* PySpark 2.4.3
-* Ray 0.7.0
-* Tensorflow 1.15.0
-* aiohttp
-* setproctitle
-* scikit-learn
-* featuretools
-* pandas 
-
 ### Usage
```
Comment (shane-huang) [inline pyzoo/zoo/zouwu/use-case/network_traffic/README.md:22]: Remove ###Usage section title

File: pyzoo/zoo/zouwu/use-case/network_traffic/README.md [hunk 4]
```diff
@@ -19,29 +19,11 @@ ie. LSTM and MTNet, to do time series forecasting. Both univariate and multivari
...
 
 #### Prepare environment
```
Comment (shane-huang) [inline pyzoo/zoo/zouwu/use-case/network_traffic/README.md:24]: Change "Prepare environment" to "Install"

**Comment by shane-huang** [comment]
Others LGTM

**Comment by yushan111** [comment]
jenkins passed http://10.239.47.210:18888/job/ZOO-PR-Doc-Generation-Test/378/

**Pull request merged**